### PR TITLE
Validate contact us form by clicking submit button

### PR DESCRIPTION
**Pull request by OleksandrDragun**
### PR Checklist

* [ ] Have you followed the guidelines in our [Dev Guide](../#readme)
* [ ] Have you lint your code locally prior to submission (`npm run lint`)?
* [ ] Have you run prettier (`npm run format`)?
* [ ] Have you checked that app can be built (`npm run build`)?
* [ ] Have you checked that the build output looks according to design (screen, mobile, desktop) in Figma (`npm run serve`)?

### CODE REVIEW

File: src/containers/ContactUsPage/ContactUsForm/constants.ts [hunk 1]
```diff
@@ -1 +1,4 @@
 export const MAX_LENGTH = 255;
+
+export const CONTACT_US_URL =
+  'https://testio--partial.sandbox.my.salesforce-sites.com/leadcapture/services/apexrest/leadservice';
```
Comment (ViktorSoroka07) [inline src/containers/ContactUsPage/ContactUsForm/constants.ts]: @AmsterGet, should we put it to env variables? What name would you suggest?

File: src/containers/ContactUsPage/ContactUsForm/ContactUsForm.tsx [hunk 2]
```diff
@@ -20,39 +20,53 @@ const getBlocksWith = createBemBlockBuilder(['contact-us-form']);
...
+          },
+        })
+          .then(() => {
+            showFeedbackForm();
+            setIsLoading(false);
+          })
+          .catch(() => {
+            showFeedbackForm();
+            setIsLoading(false);
+          });
```
Comment (AmsterGet) [inline src/containers/ContactUsPage/ContactUsForm/ContactUsForm.tsx]: It looks like we can use `.finally()` here, as long as we don't introduce a separate error state.

File: src/containers/ContactUsPage/ContactUsForm/constants.ts (hunk 1 above)
Comment (AmsterGet) [reply src/containers/ContactUsPage/ContactUsForm/constants.ts]: I guess `CONTACT_US_URL` is OK.

**Review by ViktorSoroka07 — DISMISSED** [review]

File: src/containers/ContactUsPage/ContactUsForm/ContactUsForm.tsx (hunk 2 above)
Comment (OleksandrDragun) [reply src/containers/ContactUsPage/ContactUsForm/ContactUsForm.tsx]: Done.

**Review by AmsterGet — DISMISSED** [review]

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**